docs/fsharp/language-reference/plaintext-formatting.md

```diff
@@ -0,0 +1,339 @@
+---
+title: "Tutorial: Plain Text Formatting"
+description: Learn how to use structured formatted plain text in F# applications and scripts.
+ms.date: 07/22/2020
+---
+
+# Plain Text Formatting
+
+F# allows values to be formatted as structured plain text in a compositional way.
+As a simple example, consider the following and note how the otuput has been formatted as a matrix-like display of tuples.
+
+```console
+dotnet fsi
+
+> let data = [ for i in 1 .. 3 -> [ for j in 1 .. 3 -> (i, j) ] ];;
+
+val data : (int * int) list list =
+  [[(1, 1); (1, 2); (1, 3); (1, 4); (1, 5)];
+   [(2, 1); (2, 2); (2, 3); (2, 4); (2, 5)];
+   [(3, 1); (3, 2); (3, 3); (3, 4); (3, 5)];
+   [(4, 1); (4, 2); (4, 3); (4, 4); (4, 5)];
+   [(5, 1); (5, 2); (5, 3); (5, 4); (5, 5)]]
+
+>  printfn "%A" data;;
+[[(1, 1); (1, 2); (1, 3); (1, 4); (1, 5)];
+ [(2, 1); (2, 2); (2, 3); (2, 4); (2, 5)];
+ [(3, 1); (3, 2); (3, 3); (3, 4); (3, 5)];
+ [(4, 1); (4, 2); (4, 3); (4, 4); (4, 5)];
+ [(5, 1); (5, 2); (5, 3); (5, 4); (5, 5)]]
+```
+
+Plain text formatting is activated when using the `%A` format in printf formatting strings. This has limited customizability.
+It is also used when formatting the output of values in F# interactive, when the output includes extra information and is additionally customizable.
+Plain text formatting is observable through the following:
+
+1. The default results of `x.ToString()` on F# union and record values, when `sprintf "%+A"` is used.
+
+2. The default debug text of F# union and record values (and structured values containing these), when `sprintf "%+A"` is also used.
```

Comment:
I'll be honest, i'm struggling to understand what this means.

Reply:
Well, it means that the when you look at the plaintext display of a record or union value in the debugger you are seeing the `sprintf "%A"` formatting of that value.  

This is because you're actually looking at the ToString of it

Perhaps it's not worth mentioning.  I mean ToString gets called in all sorts of ways by other tooling too

Reply:
I made some updates, please check